.changeset/rotten-dolls-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@contentful/f36-button": minor
```

Comment:
should we include all the components where you've adjusted styles?

Reply:
Done.

Reply:
I think it doesn't matter here since everything gets bumped anyway but I can add it.